core/embed/models/T3B1/vendorheader/vendor_prodtest.json

```diff
@@ -0,0 +1,22 @@
+{
+    "header_len": 512,
+    "text": "UNSAFE, FACTORY TEST ONLY",
+    "hw_model": "T3B1",
+    "expiry": 0,
+    "version": [0, 0],
+    "sig_m": 2,
+    "trust": {
+        "_reserved": 0,
+        "_dont_provide_secret": false,
```

Comment:
i believe we can drop those fields, right?

Reply:
dropped https://github.com/trezor/trezor-firmware/commit/a80f055bcf5c718cb0cb69f1b6203a005f35f22b

Reply:
i meant both `_dont_provide_secret` and `_reserved`.

right now I can't figure out which should be in the prod header, but we definitely shouldn't have `_reserved: 0` for both UNSAFE and prod headers.
the construct file has `default: 0b11111111`. all those will end up encoded as zero in the binary header. I think we want all-zero UNSAFE header, with the only `1` being `allow_run_with_secret` -- because historically, incorrectly, this was set to `1` in the old UNSAFE header.

and _presumably_ we also want these new fields to be zeroes in the prod header, so if we add a new capability, the prod headers don't automatically gain it.

Reply:
(ping in case you missed this, i forgot to unresolve the comment after adding to it)

Reply:
i think removing the _reserved everywhere is correct. 

see original discussion here https://github.com/trezor/trezor-firmware/pull/3683

and also this is how T3T1 VHs are setup

fixed https://github.com/trezor/trezor-firmware/commit/28a002a45723a813654108708ed5c57c04d2c8b9